gouttelette/data/plugins/module_utils/core.py

```diff
@@ -0,0 +1,511 @@
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is BSD licensed.
+# Modules you write using this snippet, which is embedded dynamically by Ansible
+# still belong to the author of the module, and may assign their own license
```

Comment:
Since these module_utils plugins are specific to the collection, we should move them in the plugins/module_utils directory of the final collection.